pontoon/base/templates/account_disabled.html

```diff
@@ -0,0 +1,6 @@
+{% extends "404.html" %}
+
+{% block title %}Account Disabled{% endblock %}
+{% block description %}
+Your account has been disabled. If you believe this is a mistake or need further assistance, please contact us at {{ DEFAULT_FROM_EMAIL }}.
+{% endblock %}
```

Comment:
Please add the [new line at the end of the file](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).